salt/orchestrate/aws/mitx_qa_elb.sls

```diff
@@ -1,24 +1,13 @@
-{% set type_counts = {'draft': 2, 'live': 2} %}
-{% set domains = {
-    'draft': ['studio-mitx-qa-draft.mitx.mit.edu.',
-              'mitx-qa-draft.mitx.mit.edu.',
-              'preview-mitx-qa-draft.mitx.mit.edu.',
-              'gr-qa.mitx.mit.edu.'],
-    'live': ['studio-mitx-qa.mitx.mit.edu.',
-             'mitx-qa.mitx.mit.edu.',
-             'preview-mitx-qa.mitx.mit.edu.',
-             'prod-gr-qa.mitx.mit.edu.']
-} %}
-{% set environment = 'mitx-qa' %}
+{% from "orchestrate/aws_env_macro.jinja" import VPC_NAME, VPC_RESOURCE_SUFFIX,
+ ENVIRONMENT, PURPOSE_PREFIX, subnet_ids with context %}
+{% set env_settings = salt.pillar.get('environments:{}'.format(ENVIRONMENT)) %}
+
 {% set security_groups = salt.pillar.get('edx:lb_security_groups', ['default', 'edx-mitx-qa']) %}
```

Comment:
If the value is not present in pillar then it will use the default values passed as the second parameter of the `get` call.